x/auth/ante.go

```diff
@@ -69,6 +69,8 @@ func NewAnteHandler(ak AccountKeeper, fck FeeCollectionKeeper) sdk.AnteHandler {
 			return newCtx, err.Result(), true
 		}
 
+		newCtx.GasMeter().ConsumeGas(params.TxSizeCostPerByte*sdk.Gas(len(newCtx.TxBytes())), "txSize")
+
 		if res := ValidateMemo(newCtx.GasMeter(), stdTx, params); !res.IsOK() {
```

Comment:
If we go with the route of consuming gas proportional to the tx size, then I see it as unnecessary to also charge gas for the tx's memo size.

Reply:
Agree

Reply:
agreed